src/main/scala/cromwell/engine/TaskExecutionContext.scala

```diff
@@ -0,0 +1,3 @@
+package cromwell.engine
+
+trait TaskExecutionContext
```

Comment:
Yes.  I'll add this to a comment in this file:

> TaskExecutionContext represents the static information about a specific Call's invocation.  For the LocalBackend, this would hold information like the Call's current working directory where it executes the command and a path to stdout/stderr files (which might not exist yet).  An object with this trait is available to all WDL standard library functions.  This makes it possible to implement WDL functions like `read_lines` and `write_lines`, which need have some execution context in order to work.